app/helpers/search_helper.rb

```diff
@@ -85,16 +85,17 @@ def with_notes_anchor(event, tokens)
     end
   end
 
-  def type_label(t)
-    l("label_#{t.singularize}_plural", default: t.to_s.humanize)
+  def attachment_fulltexts(event)
+    attachment_strings_for(:fulltext, event)
+  end
+
+  def attachment_filenames(event)
+    # TODO: attribute "filename" might not be available on fog?
```

Comment:
Is this a Todo that we should handle now?

Reply:
Checked it and removed it.